### PR TITLE
Chaos type add in charts

### PR DESCRIPTION
**Pull request by is-ashish**
added infra level tag

### CODE REVIEW

**Comment by ksatchit** [comment]
can we make it just "infra" v/s "infra-level"

**Comment by amitbhatt818** [comment]
DCO is getting failed @is-ashish

File: charts/generic/cpu-hog/cpu-hog.chartserviceversion.yaml [hunk 1]
```diff
@@ -22,6 +22,7 @@ spec:
   platforms:
     - GKE
   maturity: alpha
+  chaosType: infra-level
```
Comment (chandankumar4) [inline charts/generic/cpu-hog/cpu-hog.chartserviceversion.yaml]: `infra-level` -> `infra`

Comment (is-ashish) [reply charts/generic/cpu-hog/cpu-hog.chartserviceversion.yaml]: done

**Review by ksatchit — APPROVED** [review]
/lgtm

**Pull request merged**